main.go

```diff
@@ -17,8 +17,13 @@ func main() {
 	godotenv.Load()
 
 	host := os.Getenv("CHAT_HOST")
+	// Don't disregard the error returned by Atoi here
 	port, _ := strconv.Atoi(os.Getenv("CHAT_PORT"))
 
+	// You've fetched your port above as a string using os.Getenv, converted it
+	// to an integer and then used Itoa to convert it back to a string - you
+	// could just use the host + ":" + os.Getenv("CHAT_PORT") and skip all
+	// of the conversions
```

Comment:
too true. this is another instance of wanting to use the strconv functions to get that finger-memory going for Go. originally `port` and `host` were constants, and changed them up to read from `.env`